frontend/.gitignore

```diff
@@ -27,7 +27,7 @@
 # misc
 /.sass-cache
 /connect.lock
-/coverage
+/coverage/
```

Comment:
This change is unnecessary. How about just reverting change of this `.gitignore` file?